linode/manager
# feat: [UIE-8002] - DBaaS 2.0 Landing Page

## Description 📝
DBaaS 2.0 Landing Page and Empty Landing Page

## Changes  🔄
- Added Tabs to the Landing Page: Two tabs for Beta customers, labeled "Legacy Databases" and "New Databases."
- Added Logo(including dark theme): Displayed on the Empty Landing Page, in the "New Databases" tab, and in the "New Databases" single table view.
- Single Table Landing Page: Added for Beta customers without "Legacy Databases" but with "New Databases," and for new customers with only "New Databases."
- New "Plan" Column: Added to the "New Databases" table.
- Added Info Banner: Included on the landing page.

## Target release date 🗓️
9/9/24

## Preview 📷
**Include a screenshot or screen recording of the change**

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-08-23 at 2 21 42 PM](https://github.com/user-attachments/assets/2ae8e205-fcf9-4841-98f5-6c17c6c067bc) | ![Screenshot 2024-08-23 at 2 20 46 PM](https://github.com/user-attachments/assets/bfe1e9c5-aa01-4835-bdda-cfb490feb9bb) |
|![Screenshot 2024-08-23 at 2 17 50 PM](https://github.com/user-attachments/assets/72aae648-bdf3-42e9-a25d-6e742c8db589) | ![Screenshot 2024-08-23 at 2 18 05 PM](https://github.com/user-attachments/assets/0338b717-5d38-4257-b5f9-f8b024ba8bc1)|
| |![Screenshot 2024-08-23 at 5 40 13 PM](https://github.com/user-attachments/assets/ffa7c3b4-4947-46d7-b3b0-286d8f22caa9)|
| |![Screenshot 2024-08-23 at 2 22 56 PM](https://github.com/user-attachments/assets/571ba607-dd92-4efa-9b15-e955cc38f5b3)|

## 
How to test 🧪

**Note:
To test all beta features, beta must be enabled, and the user must have the "Managed Databases Beta" account capability.**

### Prerequisites
(How to setup test environment)
Enable DatabaseV2Beta flag

### Verification steps
(How to verify changes)

1. Navigate to the Databases section.
2. For new customers (with no legacy or new databases), the Landing Page will display an Empty Landing Page with a Logo.
3. For customers with Legacy Databases, the Landing Page will show tabs for "Legacy Databases" and "New Databases," along with an info banner.
4. When "New Databases" tab is selected, the page will display a table with an additional "Plan" column and a logo below the table. The "Configuration" column will be renamed to "Nodes." The table will be empty if no new databases are present.
5. For customers without Legacy Databases but with New Databases, the Landing Page will show a single table with a "Plan" column and the "Configuration" column renamed to "Nodes," an info banner, and a Logo below the table.
6. If the DatabaseV2Beta flag is disabled, the Landing Page will display a single table with no changes.

## As an Author I have considered 🤔

*Check all that apply*
